src/components/FlightTicket/FlightTicket.module.css

```diff
@@ -0,0 +1,72 @@
+.ticketRed {
+  display: flex;
+  flex-direction: column;
+  background-color: lightgrey;
+  width: 270px;
+  height: fit-content;
+  font-family: "Roboto", sans-serif;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  background-color: #ffdada;
+  border: 3px solid #f7baba;
+}
+
+.ticketBlue {
+  display: flex;
+  flex-direction: column;
+  background-color: lightgrey;
+  width: 270px;
+  height: fit-content;
+  font-family: "Roboto", sans-serif;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  background-color: #d9e4ff;
+  border: 3px solid #b3c3ec;
+}
+
+.row1 {
+  /* background-color: rgb(237, 42, 42); */
+  font-size: 0.8rem;
+  padding-top: 5px;
+  padding-bottom: 10px;
+}
+
+.row2 {
+  /* background-color: rgb(211, 56, 189); */
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  font-size: 3rem;
+  font-weight: 600;
+}
+
+.row3 {
+  /* background-color: rgb(211, 123, 200); */
+  display: flex;
+  justify-content: space-between;
+  font-size: 0.6rem;
+  font-weight: 900;
+}
```

Comment:
Can we rename `row1`, `row2` and `row3` to something that describes the row? Maybe `dateRow`, `airportsRow` and `detailsRow`?